qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -889,38 +889,23 @@ class AcquisitionController(Instrument):
         _alazar: a reference to the alazar instrument driver
     """
 
-    def __init__(self, name, alazar_name, **kwargs):
-        """
-        Args:
-            alazar_name: The name of the alazar instrument on the server
-        """
-        super().__init__(name, **kwargs)
-        self._alazar = self.find_instrument(alazar_name,
-                                            instrument_class=AlazarTech_ATS)
+    def __init__(self, *args, **kwargs):
```

Comment:
this is not needed, right?

Reply:
I think it might be, when using the interface as a mixin.